docs/contributors_guide/maintenance/iso_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -0,0 +1,261 @@
+   cd simp-core/src
+   bundle update
+   bundle exec rake pkg:rpm[epel-6-x86_64]
+   bundle exec rake pkg:rpm[epel-7-x86_64]
+
+Verify simp-core unit tests pass
+--------------------------------
+
+This check verifies that the ``simp-core`` unit tests have succeeded
+in `TravisCI`_.
+
+   * Navigate to the project's TravisCI results page and verify the
+     tests for the development branch to be tagged and released have
+     passed.  For our project, this page is
+     https://travis-ci.org/simp/simp-core/branches
+
+     .. IMPORTANT::
+
+        If the tests in TravisCI fail, you **must** fix them before
+        proceeding.  The automated release procedures will only
+        succeed, if the unit tests succeed in TravisCI.
+
+Verify ISOs can be created
+--------------------------
+
+This check verifies that SIMP ISOs for CentOS 6 and CentOS 7 can be
+built from the local simp-core RPM and RPMs pushed to packagecloud.
+
+* FIXME - The magic happens here....Trevor's build script?
+
+Verify SIMP ISO boot options work
+---------------------------------
+
+This hefty check verifies that a server booted from the SIMP ISO can
+be bootstrapped for the 'simp' scenario and following boot options:
+
+* Using default boot option
+* Using disk encryption boot option
+* Using FIPS disabled boot option
+* Using disk encryption and FIPS disabled boot options
+* Using simp-prompt option
+* Using simp-prompt and disk encryption boot options
+* Using simp-prompt and FIPS disabled boot options
+* Using simp-prompt, disk encryption, and FIPS disabled boot options
+* Using linux-min boot option
+* Using linux-min and disk encryption boot options
+* Using linux-min and FIPS disabled boot options
+* Using linux-min, disk encryption, and FIPS disabled boot options
+
+For the default boot option and and FIPS disabled boot option
+test cases, the `simp-packer`_ project is the easiest way to
+verify a SIMP VM can be booted from the ISO and bootstrapped.  Otherwise,
+the check has to be done manually:
+
+* Boot a VM with the SIMP ISO
+* Select the appropriate boot options
+* Once the server boots, login to the server as root
+* Bootstrap the system
+
+  .. code-block:: bash
+
+     simp config
+     simp bootstrap
+     reboot
+
+* Login to the server as root and run ``puppet agent -t`` until the
+  results are stable
+* Verify the server is/is not in FIPS mode by inspecting `/proc/sys/crypto/fips_enabled`
+* Verify the appropriate disk is/is not encrypted by executing
+
+  .. code-block:: bash
+
+     blkid
+
+* Verify the appropriate disk partitioning
+
+  .. code-block:: bash
+
+     lsblk
+
+Verify component interoperability
+---------------------------------
+
+This check verifies with ``pupmod-simp-simp`` acceptance tests that this
+aggregation of module versions interoperate. (These tests provide
+extensive, cross-component, integration tests.)
+
+.. NOTE::
+   If ``pupmod-simp-simp`` acceptance tests have effectively 
```

Comment:
@op-ct  Absolutely we need integration tests.  We also need to automate more of the manual steps in this document.